### PR TITLE
Update  UI

### PR DESCRIPTION
**Pull request by songivan00**
Update MainWindow to support TaskListPanel

### CODE REVIEW

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -15,13 +15,16 @@
 import seedu.address.commons.util.StringUtil;
 import seedu.address.logic.Logic;
 import seedu.address.logic.LogicManager;
+
```
Comment (songivan00) [inline src/main/java/seedu/address/MainApp.java:18]:
```suggestion
```

File: src/main/java/seedu/address/MainApp.java [hunk 2]
```diff
@@ -15,13 +15,16 @@
...
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
 import seedu.address.model.ReadOnlyAddressBook;
 import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.TaskList;
 import seedu.address.model.UserPrefs;
+import seedu.address.model.ModelManager;
+import seedu.address.model.Model;
 import seedu.address.model.util.SampleDataUtil;
+
```
Comment (songivan00) [inline src/main/java/seedu/address/MainApp.java:27]:
```suggestion
```

File: src/main/java/seedu/address/MainApp.java [hunk 3]
```diff
@@ -15,13 +15,16 @@
...
 import seedu.address.logic.LogicManager;
+
 import seedu.address.model.AddressBook;
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
 import seedu.address.model.ReadOnlyAddressBook;
 import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.TaskList;
 import seedu.address.model.UserPrefs;
+import seedu.address.model.ModelManager;
```
Comment (songivan00) [inline src/main/java/seedu/address/MainApp.java:24]:
```suggestion
import seedu.address.model.Model;
```

File: src/main/java/seedu/address/MainApp.java [hunk 4]
```diff
@@ -15,13 +15,16 @@
...
+import seedu.address.model.Model;
```
Comment (songivan00) [inline src/main/java/seedu/address/MainApp.java:25]:
```suggestion
import seedu.address.model.ModelManager;
```

**Review by songivan00 — COMMENTED** [review]
Change code for checkstyle

**Review by angkl0 — APPROVED** [review]
LGTM!

**Pull request merged**